frontend/src/pages/projects/screens/detail/storage/__tests__/useProjectPvcs.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { k8sListResourceItems } from '@openshift/dynamic-plugin-sdk-utils';
+import { act } from '@testing-library/react';
+import { mockPVCK8sResource } from '~/__mocks__/mockPVCK8sResource';
+import { standardUseFetchState, testHook } from '~/__tests__/unit/testUtils/hooks';
+import useProjectPvcs from '~/pages/projects/screens/detail/storage/useProjectPvcs';
+
+jest.mock('@openshift/dynamic-plugin-sdk-utils', () => ({
+  k8sListResourceItems: jest.fn(),
+}));
+
+const k8sListResourceItemsMock = k8sListResourceItems as jest.Mock;
+
+describe('useProjectPVCs', () => {
+  it('should return dashboard PVCs', async () => {
+    const mockedPVC = { data: mockPVCK8sResource({}) };
+    k8sListResourceItemsMock.mockReturnValue(Promise.resolve(mockedPVC));
+
+    const renderResult = testHook(useProjectPvcs)('namespace');
+    expect(k8sListResourceItemsMock).toHaveBeenCalledTimes(1);
```

Comment:
I was just thinking about how we can make these tests a bit more robust. 
There's nothing wrong with the current test and it aligns with what we've done for other tests (we have several tests that only go as far as to test that the api was invoked).

I think we should not only assert that `k8sListResourceItemsMock` was called, but also what it was called with. This will ensure that we're calling it with the correct model and query params.
Can you add an assertion here:

```
expect(k8sListResourceItemsMock).toHaveBeenCalledWIth(...);
```